Cargo.toml

```diff
@@ -46,7 +46,8 @@ regex = "1"
 config = "0.9"
 serde = { version = "1.0", features = ["derive"] }
 serde_yaml = "0.8"
-tox = "0.0.9"
+tox = { git = "https://github.com/tox-rs/tox.git" }
```

Comment:
This pr should contain only Dockerfile. Or is there a reason for this?

Reply:
There is a reason for that. There are issues with old sodiumoxide, without updates this PR won't pass CI.

Still we have to update deps in another PR